pyproject.toml

```diff
@@ -28,21 +28,26 @@ nml = "nannyml.cli.cli:cli"
 nannyml = "nannyml.cli.cli:cli"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
-pandas = [
-    {version="^1.5", python=">=3.8,<3.12"}
-]
+python = ">=3.9,<3.13"
+
+numpy = "^2.0"
+
+scipy = "^1.13"
+
+pandas = "^2.2"
+
+scikit-learn = "^1.6"
+
+pyarrow = "^18.1"
```

Comment:
this is also the latest version released nov, I think you should support it from atleast 13.0 or 14.0, those versions were released just last year.